.gitignore

```diff
@@ -18,6 +18,8 @@ coverage.xml
 venv
 env2
 env3
+boto3_venv/
```

Comment:
We should remove this since it isn't a standard virtual environment name.
I'll link a guide for ignoring local files:
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file